frontend/simple/js/state.js

```diff
@@ -48,15 +56,22 @@ const mutations = {
     // we've successfully received it back, so remove it from expectation pending
     const index = state.pending.indexOf(contractId)
     state.pending.includes(contractId) && state.pending.splice(index, 1)
+    // calling this will make pubsub subscribe for events on `contractId`!
+    sbl('vuex/v1/events/emit', 'contractsModified', {add: contractId})
```

Comment:
No, it's not async, see the implementation in the sbl file. sbl calls can be async or normal, it all depends on the functions that were registered